### ISSUE TITLE
Add new funny algorithms!

### ISSUE THREAD

**Issue by OtacilioN** [ReciHub/FunnyAlgorithms#9]
[Check our Contributing Guide](https://github.com/ReciHub/FunnyAlgorithms/blob/master/CONTRIBUTING.md) and add new funny algorithms to the repository.

You can add any algorithm that you consider fun, open a pull request and explain why it is fun!

**Comment by bhagatparwinder** [comment]
@OtacilioN Added a new algorithm in JS 😄

**Comment by OtacilioN** [comment]
Awesome, thanks for contributing!

**Comment by rmehri01** [comment]
@OtacilioN I added bogoSort, is there anything wrong with it?

**Comment by Habernet** [comment]
Working on adding multiple algorithms in JS!

**Comment by somyaguglani** [comment]
i referred to wrong issue at first , i corrected it just now..
 please accept my PR

**Comment by Thuvan96** [comment]
Can i join ??

**Comment by OtacilioN** [comment]
Yes of course @Thuvan96 everyone is welcome here :sparkling_heart:

**Comment by somyaguglani** [comment]
@OtacilioN could you assign us the issues we opened?  Otherwise our contribution becomes invalid for hacktoberfest

**Comment by OtacilioN** [comment]
Hey @somyaguglani don't worry, your hacktoberfest contributions are already counting, and you can check it in https://hacktoberfest.digitalocean.com/profile

Now hacktoberfest has a 7 day period to validate the pull request, where maintainers can mark spamming pull requests as "invalid", and you can see in [this pull request](https://github.com/ReciHub/FunnyAlgorithms/pull/198) so in that case, the pull request will not count.

**Comment by somyaguglani** [comment]
@OtacilioN thank you for explaining , really helpful

**Comment by toclean** [comment]
Just ported Hot Potato to Typescript!! Yay Typescript!

**Comment by XVR147** [comment]
Just added Doomsday algorithm in Haskell with #450 altho it's a bit noobish 🤕

**Comment by roh-it** [comment]
Added binary search https://github.com/ReciHub/FunnyAlgorithms/pull/470

**Comment by somyaguglani** [comment]
Added #395 and #396 new algos for linkedlists

**Comment by subhankar01** [comment]
Added spiral and WaterOverflow problem in Java #471

**Comment by subhankar01** [comment]
Added MagicSquare program in java

**Comment by gmontalvo404** [comment]
Added two algorithms in java (Palindrome and Calculadora)

**Comment by naruto678** [comment]
HI everyone .. Can i join. I want to add graph algorithms .. It will contain dfs, bfs, djikstra and flood filling and others all implemented in java

**Comment by subhankar01** [comment]
Yeah sure !!

**Comment by naruto678** [comment]
Added dfs

**Comment by happycoder0011** [comment]
Can I add pigeon sort in C++ and Js?

**Comment by subhankar01** [comment]
Sure

**Comment by Vivekumar01** [comment]
I would like to add an ellipse drawing algorithm in python. Please assign this issue to me.

**Comment by AyushCSE04** [comment]
Can i add insertion ,deletion algos for arrays ?

**Comment by pathakharsh123** [comment]
my pull request is pending for N-Queens code in java kindly approve it

**Comment by pjyoti121** [comment]
I want to join this please assign me

**Comment by TrlRizu** [comment]
Heyy! I opened a pull request for a cool cipher (Beaufort Cipher). Hope you can merge it! https://github.com/ReciHub/FunnyAlgorithms/pull/1101 Have a nice day!

**Comment by Yashpsct010** [comment]
please assign me this issue

**Comment by PS-11** [comment]
Can I add some algos? @OtacilioN

**Comment by rishabh-i13** [comment]
Please assign me this issue , i will be adding some more algos.

**Comment by Abhipsa05** [comment]
Please assign this issue to me, I would like to add some algos.

**Comment by zykuriku** [comment]
I'd like to add an algo.. could this issue be assigned to me please?

**Comment by NickzArora** [comment]
Added a v simple Bubble Sort in c# https://github.com/ReciHub/FunnyAlgorithms/pull/1172

**Comment by Akangkha** [comment]
Added "Convey game of life" alg in c and js for contributing in hactoberfest '23
c#1180

**Comment by Nandita-rawat** [comment]
is this issue still open for hactoberfest'23 ? would like to add a fun algo

**Comment by Kedarnath-Rothe** [comment]
Please assign me this issue

**Comment by NickzArora** [comment]
@OtacilioN Created BlackJack Game in c#. -https://github.com/ReciHub/FunnyAlgorithms/pull/1305
